src/styles/blocks/catalog.scss

```
@@ -17,6 +17,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transorm $trstn-drtn,
```

```suggestion
    transition: transform $trstn-drtn,
```
Should fix why transition not applied

---

what is the strange name `$trstn-drtn`?)

---

thank you for the corrections,
mistakes have been fixed

---

yep, name should be always understandable